Description: Add support for steelseries Gaming headphones
Origin: upstream, https://cgit.freedesktop.org/pulseaudio/pulseaudio/commit/?id=3454c19f3c277d5d0099f17e7ebf5d2005afa4b0
Bug-Ubuntu: https://launchpad.net/bugs/1758736
Author: Dave Chiluk <chiluk@ubuntu.com>

-----------------------------------------------
commit 3454c19f3c277d5d0099f17e7ebf5d2005afa4b0
Author: Zakhary Husak <zakharhusak@gmail.com>
Date:   Wed Oct 24 04:54:56 2018 +0000

    alsa-mixer: Add support for 2018 Arctis 7

Index: pulseaudio-12.2/src/modules/alsa/mixer/profile-sets/90-pulseaudio.rules
===================================================================
--- pulseaudio-12.2.orig/src/modules/alsa/mixer/profile-sets/90-pulseaudio.rules
+++ pulseaudio-12.2/src/modules/alsa/mixer/profile-sets/90-pulseaudio.rules
@@ -107,9 +107,11 @@ ATTRS{idVendor}=="041e", ATTRS{idProduct
 ATTRS{idVendor}=="0bda", ATTRS{idProduct}=="4014", ENV{PULSE_PROFILE_SET}="dell-dock-tb16-usb-audio.conf"
 ATTRS{idVendor}=="1038", ATTRS{idProduct}=="1250", ENV{PULSE_PROFILE_SET}="steelseries-arctis-5-usb-audio.conf"
 
+# ID 1038:12ad is for the 2018 refresh of the Arctis 7.
 # ID 1038:1294 is for Arctis Pro Wireless (which works with the Arctis 7
 # configuration).
 ATTRS{idVendor}=="1038", ATTRS{idProduct}=="1260", ENV{PULSE_PROFILE_SET}="steelseries-arctis-7-usb-audio.conf"
+ATTRS{idVendor}=="1038", ATTRS{idProduct}=="12ad", ENV{PULSE_PROFILE_SET}="steelseries-arctis-7-usb-audio.conf"
 ATTRS{idVendor}=="1038", ATTRS{idProduct}=="1294", ENV{PULSE_PROFILE_SET}="steelseries-arctis-7-usb-audio.conf"
 
 LABEL="pulseaudio_end"
